app/Livewire/Project/Application/Previews.php

```
@@ -73,35 +93,52 @@ public function save_preview($preview_id)
-                // Check for domain conflicts if not forcing save
-                if (! $this->forceSaveDomains) {
-                    $result = checkDomainUsage(resource: $this->application, domain: $preview->fqdn);
-                    if ($result['hasConflicts']) {
-                        $this->domainConflicts = $result['conflicts'];
-                        $this->showDomainConflictModal = true;
-                        $this->pendingPreviewId = $preview_id;
-
-                        return;
+
+            if (! $preview) {
+                throw new \Exception('Preview not found');
```

PHPMD (2.15.0)

Missing class import via use statement (line '98', column '27'). (undefined)

(MissingImport)